linkml_model/model/schema/meta.yaml

```diff
@@ -1420,6 +1420,39 @@ slots:
       - BasicSubset
       - ObjectOrientedProfile
 
+  array_info:
+    domain: slot_definition
+    range: array_info_expression
+    inherited: true
+    description: coerces the value of the slot into an array and defines the dimensions of that array
+    status: testing
+
+  dimensions_info:
```

Comment:
hopefully not annoying naming question, why not just `array` and `dimensions` here?

Reply:
(we did change it to that)